src/main/target/common_fc_post.h

```diff
@@ -72,11 +72,11 @@
 #undef USE_TELEMETRY_CRSF
 #endif
 
-#if !defined(USE_SERIALRX_SPEKTRUM)
+#if !defined(USE_SERIALRX_JETIEXBUS)
 #undef USE_TELEMETRY_JETIEXBUS
 #endif
 
-#if !defined(USE_SERIALRX_JETIEXBUS)
+#if !defined(USE_SERIALRX_SPEKTRUM)
 #undef USE_TELEMETRY_SRXL
```

Comment:
This `#undef` should be moved to the section below with other undefs depending on `#ifndef USE_SERIALRX_SPEKTRUM `

Reply:
Good idea